view/frontend/templates/checkout/success.phtml

```diff
@@ -29,8 +29,8 @@
     <script type="text/javascript">
         require([
             'jquery',
-            'adyenCheckout3101'
-        ], function ($, AdyenCheckout) {
+            'Adyen_Payment/js/bundle'
+        ], function ($, adyenComponent) {
```

Comment:
This was added for Swish, back then it was the latest version so I think we can replace it with the bundle

Reply:
@acampos1916 I think so yes, also for multibanco